pkg/odo/cli/component/list.go

```diff
@@ -81,12 +114,19 @@ func (lo *ListOptions) Validate() (err error) {
 		return odoutil.ThrowContextError()
 	}
 	return nil
+
 }
 
 // Run has the logic to perform the required actions as part of command
 func (lo *ListOptions) Run() (err error) {
 
+	// --path workflow
+
 	if len(lo.pathFlag) != 0 {
+
+		if experimental.IsExperimentalModeEnabled() {
```

Comment:
Am I missing something or was this moved from somewhere else?

Reply:
component list didn't support experimental mode yet

Reply:
the --path flag support for experimental mode will be added as a follow up PR as the changes in this PR are already quite wide spread already